contrib/test/integration/main.yml

```diff
@@ -25,15 +25,15 @@
     - name: clone build and install kubernetes
       include: "build/kubernetes.yml"
       vars:
-        k8s_git_version: "release-1.18"
+        k8s_git_version: "release-1.19"
         k8s_github_fork: "kubernetes"
         crio_socket: "/var/run/crio/crio.sock"
       when: "(cgroupv2 is undefined) or (cgroupv2 == False) | bool"
 
     - name: clone build and install kubernetes for cgroup v2
       include: "build/kubernetes.yml"
       vars:
-        k8s_git_version: "master"
```

Comment:
What is the reason that we do not stick to the master here?

Reply:
Yeah, I wasn't entirely sure about this one. In 1.18 we were using @giuseppe's branch. I was testing out if this works with the release-1.19 branch, and looks like it does. But if we want to keep cgroupv2 pointing to master, I will revert it here.

Reply:
Right now they're still the same with some slight delay, but if we apply Code Thaw (in some weeks) in Kubernetes, then we will not be able to test the latest master changes any more. I think we could simply stick to master, WDYT?

Reply:
Sure, fixed!